### DISCUSSION TITLE
Compatibility with Raspberry Pi Zero W

### DISCUSSION THREAD

**Discussion by AndreiGr** [ketilmo/balena-ads-b#55] [Q&A] [answered]
Hi guys,

Has anyone tried to run this code and feed the data using a Raspberry Pi Zero W?

I'm asking because the Zero W is in the list of compatible devices for PiAware, but not in the list here.

All help is appreciated,
Andrei

**Comment by ketilmo** [comment, marked as answer]
Hi @AndreiGr,

Thanks for reaching out. The balena-ads-b solutions worked on Raspberry Pi Zeros previously. Due to the number of different services that are being fed currently, it's no longer working out of the box. But by customizing the setup and removing some of the services that are being fed, it should definitely be able to get it to work. Let us know if you give it a go. :)
